src/main/java/software/amazon/qldb/QldbDriverBuilder.java

```diff
@@ -168,4 +172,23 @@
      * @return This builder object.
      */
     QldbDriverBuilder readAhead(int readAhead, ExecutorService executorService);
+
+    /**
+     * <p>Specify the http client builder that should be used for making http requests.</p>
+     *
+     * <p>Note that if maximum connections is set in the http client, it should be equal or greater than
+     * {@link #maxConcurrentTransactions(int)} to avoid connection contentions and poor performance.</p>
+     *
+     * <p>If no parameter is specified then a default value would be used.</p>
+     *
+     * <p>
+     *     The httpClientBuilder is optional.
+     * </p>
```

Comment:
I think this part is unneeded as it just repeats the above

Reply:
We have "The `field` is optional/mandatory" for every field https://github.com/awslabs/amazon-qldb-driver-java/blob/3e7f5a4f3498513fee9cd337b8a5f14c941bca9b/src/main/java/software/amazon/qldb/QldbDriverBuilder.java#L117